scenariolib/event_pageview.go

```diff
@@ -17,6 +17,7 @@ import (
 // that the user visited a page that was returned as a result from a search.
 // The view event sent will contain {contentIdKey: "@pageViewField", contentIdValue: result['pageViewField']}
 type ViewEvent struct {
+	DocNo     	  int                    `json:"docNo,omitempty"`
```

Comment:
No for this change, what you should have if you were using the `docNo` is this:
```
ClickRank     int                    `json:"docNo"`
```

With this, you pass `docNo` as the json param key, but the attribute is being mapped to `ClickRank`

Reply:
ClickRank is already mapped with:
```
ClickRank     int                    `json:"clickRank"`
```

I wanted to have something like `json:"clickRank OR docNo"` but have not found anything.

The question is: Do you *really* want to use "docNo" or do you prefer  "clickRank".

If you want to revert back to "docNo", let's do this.

But if you prefer "clickRank", then the changes here are what is required to deprecate the old value and propose the new one

Reply:
Well I was trying to be non breaking to the json config since this is pretty much the public "api" version of the bot. So if you check in the click events, `json:"docNo"` is mapped to clickRank.

But I must've forgotten about this one here. So yes I would just revert back to 
```
ClickRank     int                    `json:"docNo"`
```